app/controllers/api/validations_controller.rb

```diff
@@ -1,9 +1,16 @@
 module Api
   class ValidationsController < ApiController
+    # rubocop:disable Metrics/MethodLength
```

Comment:
Well, I have to object to this change then.

If you disagree with the rules, you should work with the team to agree and implement a new set (or update the current one), not disable them ad-hoc as they get in your way. This is no different than disabling tests and committing them that way because you don't like the result (which, in case it's not clear, is also something one should not do).

Disabling style rules (or tests) ad-hoc also sets a bad precedent, and a bad example to others: you may know when to (and not to) disable them, and why, but will the person reading your code? What about a junior developer on the team?

Let's discuss this in person on Monday.